user_guide_src/source/dbmgmt/index.rst

```diff
@@ -7,7 +7,7 @@ CodeIgniter comes with tools to restructure or seed your database.
 .. toctree::
     :titlesonly:
 
-    Database Manipulation with Database Forge <forge>
+    Database Forge <forge>
```

Comment:
I think "Database Forge" is a bit difficult to understand. The word "Forge" is difficult.
So "Database Manipulation with Database Forge" is better.
Why don't we change the page title instead?

Reply:
@kenjis I would prefer to shorten the menu entry. Some options are:
Database Management
Database and Table Management
Database Manipulation
Database Operations

Reply:
I don't think these are good. They are ambiguous or too broad.
"Database Forge" is better than them.